### PR TITLE
Updated virt screenshots + added new security option

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: xml/libvirt_configuration_gui.xml
Comment (aginies) [inline xml/libvirt_configuration_gui.xml:1]: you can add this requires a Q35 mlachine

**Review by aginies — APPROVED** [review]

File: xml/libvirt_configuration_gui.xml [hunk 1]
```diff
@@ -282,6 +282,16 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+            <term>Enable launch security</term>
+            <listitem>
+              <para>
+                If the &vmhost; supports AMD-SEV technology, activating this
+                option enables secured guest with memory encryption. This option
```
Comment (dariavladykina) [inline xml/libvirt_configuration_gui.xml:290]: It's difficult to parse this sentence for me :-) 
Does it mean that "activating the option enables secure guest access and/with memory encryption"? 
Or "activating the option provides memory encryption to the secure guest"?

Comment (aginies) [reply xml/libvirt_configuration_gui.xml:290]: In fact, this is more complex. This checkbox option is ~10% of what needs to be done to support memory encryption. this Memory encryption in virt-manager is useless as it doesn't do all the requested actions to be able to support it. Probably we should point  to the doc related to this tech: https://documentation.suse.com/sles/15-SP4/single-html/SLES-amd-sev/#article-amd-sev

Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:290]:
```suggestion
                option enables secured guest with encrypted memory. This option
```

File: xml/libvirt_configuration_gui.xml [hunk 2]
```diff
@@ -282,6 +282,16 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+            <term>Enable launch security</term>
+            <listitem>
+              <para>
+                If the &vmhost; supports AMD-SEV technology, activating this
+                option enables secured guest with encrypted memory. This option
```
Comment (tbazant) [inline xml/libvirt_configuration_gui.xml:290]:
```suggestion
                option enables a secured guest with encrypted memory. This option
```

**Review by dariavladykina — APPROVED** [review]

File: xml/libvirt_configuration_gui.xml (hunk 1 above)
Comment (tbazant) [reply xml/libvirt_configuration_gui.xml]: done

**Pull request merged**